massa-execution-worker/src/tests/wasm/sources/datastore_manipulations/main.ts

```diff
@@ -0,0 +1,16 @@
+/** ****************************
+ * Bytecode to send to the massa network that push the `helloworld`
+ * smartcontract.
+ *
+ * N.B. The entry file of your Massa Smart Contract program should be named
+ * `src/main.ts`, the command `yarn bundle` will produce an `build/main.wasm`
+ * which is ready to be send on Massa network node!
+ **/
+
+import {Storage} from "@massalabs/massa-sc-std";
```

Comment:
Would be better to use massa-as-sdk now...

Reply:
Opened an issue to update all : #3082

Reply:
I've added a comment in #3082 but there was already an issue opened that is very similar: https://github.com/massalabs/massa/issues/3045